code/modules/mob/living/carbon/human/human_update_icons.dm

```diff
@@ -714,9 +714,72 @@ There are several things that need to be remembered:
 		// SKYRAT EDIT END
 		overlays_standing[BACK_LAYER] = back_overlay
 	apply_overlay(BACK_LAYER)
-
+// BUBBER EDIT START
+/obj/effect/abstract/held_tk_effect
```

Comment:
Can't you put these in a seperate bubber file? I know  proc edits beneath you can't fully modularize, but this could be moved over

Reply:
Sure